.github/workflows/release/Dockerfile.ubuntu-18.04

```diff
@@ -0,0 +1,78 @@
+ARG GO_VERSION
+ARG GO_IMAGE=golang:${GO_VERSION}
+FROM --platform=$BUILDPLATFORM $GO_IMAGE AS go
+
+FROM --platform=$BUILDPLATFORM ubuntu:18.04 AS base
```

Comment:
Any reason to use such an old version of Ubuntu? Can we just use 20.04?

Reply:
Because that's what we are already using.
These are dynamically linked binaries and I didn't want to mess with that.